packages/cudos-test/_run-tests.sh

```diff
@@ -0,0 +1,53 @@
+source ./packages/cudos-test/_vars.sh
```

Comment:
Do you think `cudos-tests` or `cudos-testing` will be a better folder name than `cudos-test`?